src/products/components/ProductListPage/ProductListPage.tsx

```diff
@@ -122,59 +134,93 @@ export const ProductListPage: React.FC<ProductListPageProps> = props => {
 
   return (
     <ListPageLayout>
-      <TopNav withoutBorder title={intl.formatMessage(sectionNames.products)}>
-        <Box display="flex" alignItems="center" gap={5}>
-          {hasLimits(limits, "productVariants") && (
-            <Text variant="caption">
-              {intl.formatMessage(
-                {
-                  id: "Kw0jHS",
-                  defaultMessage: "{count}/{max} SKUs used",
-                  description: "created products counter",
-                },
+      <TopNav
+        withoutBorder
+        isAlignToRight={false}
+        title={intl.formatMessage(sectionNames.products)}
+      >
+        <Box
+          __flex={1}
```

Comment:
**thought:** we should consider adding common flex-grows or just `flex` values to sprinkles

Reply:
IMO we should